tests/unit/service/test_storage.py

```diff
@@ -1,6 +1,8 @@
 from logging import getLogger
-from os import listdir, remove, removedirs
+from os import listdir, remove, removedirs, path
 from random import randint
+from shutil import rmtree
+from tempfile import mkdtemp
```

Comment:
The gateway should automatically create required folders, you don't need to do this manually.